libraries/Microsoft.Bot.Builder.Ai/QnAMaker.cs

```diff
@@ -76,6 +88,18 @@ public class QnAMakerOptions
         public string KnowledgeBaseId { get; set; }
         public float ScoreThreshold { get; set; }
         public int Top { get; set; }
+        public Metadata[] StrictFilters { get; set; }
+        public Metadata[] MetadataBoost { get; set; }
+    }
+
+    [Serializable]
+    public class Metadata
```

Comment:
Perhaps a silly question, but the QnA Maker JSON element for this is "metadata" (lowercase m). When this is serialized, will that matter? I see this is covered via the JsonProperty attributes on name/value, but I'm not sure about the class name.

Reply:
Just following up on this point. We only serialise the contents of the Metadata object (hence needing the json attributes on the name / value). When building up the request for the QnA Maker we explicitly build the object using the name "metadata". If this were to change then we would need the json property on the metadata object, so I might add it for good measure in a follow up PR when I add some additional tests mentioned above.